### PR TITLE
docs(enterprise/changelog) clarify request-validator

### PR DESCRIPTION
**Pull request by gszr**
## Summary

Clarify request validator schema support.

### CODE REVIEW

File: app/enterprise/changelog.md [hunk 1]
```diff
@@ -129,9 +129,10 @@ for details.
     parameter. Now you can provide status code ranges and single status codes
     together (e.g., `201-204,401`)
   - **Request Validator** (_NEW_):
-    - Validate the request body against a JSON schema; if the payload is not
+    - Validate the request body against a schema; if the payload is not
     valid JSON or if it doesn't conform to the schema, the plugin returns a 400
```
Comment (h3xar0n) [inline app/enterprise/changelog.md]: shouldn't we remove `is not valid JSON` as well?

Comment (gszr) [reply app/enterprise/changelog.md]: This PR only clarifies that the schema format used is not JSON schema (http://json-schema.org); but the request body still is validated, both syntactically (valid JSON) and semantically (conforms to the schema spec), so we shoudn't remove that.

Comment (h3xar0n) [reply app/enterprise/changelog.md]: Understood, thank you for clarifying.

**Comment by gszr** [comment]
Thank you @h3xar0n!

**Pull request merged**